console/console-init/ui/src/modules/address-detail/AddressDetailPage.tsx

```diff
@@ -38,13 +38,13 @@ export default function AddressDetailPage() {
     () => (
       <Breadcrumb>
         <BreadcrumbItem>
-          <Link id="ad-page-link-home" to={"/"}>
+          <Link id="adr-details-home-link" to={"/"}>
```

Comment:
A typo here.
`adr` -> `addr`
Just to keep stuffs uniform.

Reply:
Thanks. I made a typo there. Resolved :)